Cake.BenchmarkDotNet/Cake.BenchmarkDotNet.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net5.0;net6.0</TargetFrameworks>

```

Comment:
Can we swap this to just netcore3.1 and only have a single target? Full framework isn't supported with Cake 2.X anyways.

Reply:
My intent was to pull in the latest applicable `Cake.Core` version for each of the listed frameworks. The `netstandard2.0` target would pull in the latest `Cake.Core v1.3.0` that still supports legacy frameworks like `net48` and `netcoreapp3.1`. The `net5.0` and `net6.0` are necessary to be able to pull in the absolute latest `Cake.Core v2.2.0` which supports the latest modern frameworks. I was concerned about dropping `netstandard2.0` since I am not sure if there are teams that would not be able to upgrade because of this breaking change. I am happy to talk this through with you if you would like. You know how to reach me. 😄

Reply:
Given that cake has deprecated support for full framework and already broke it, I think I am ok with the breaking change here.

Reply:
Fixed: 17c1b9ee130258f54733a5192d916451eebb022f